react-native-async-storage/async-storage
# Migrate Windows test app to `react-native-test-app`

## Summary

Migrates Windows test app to `react-native-test-app`.

## Remaining Work

- [x] ~~Figure out why AsyncStorage isn't autolinked~~ Resolution: Requires bumping `react-native-windows` to 0.63
- [x] Wait for Windows template fix: https://github.com/microsoft/react-native-test-app/pull/274

## Test Plan

1. Build Windows example app:
    ```
    yarn
    yarn install-windows-test-app -p example/windows
    start example/windows/ReactTestApp.sln
    yarn start:windows
    ```
2. Make sure the app works as expected

## Screenshots

Since we're also bumping `react-native`, we should also make sure we haven't broken other platforms.

| Android | iOS | macOS | Windows |
|:-:|:-:|:-:|:-:|
| ![Screenshot_1611676595](https://user-images.githubusercontent.com/4123478/105869395-7b973500-5ff7-11eb-9e32-f83cbc7a8127.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-01-26 at 16 46 56](https://user-images.githubusercontent.com/4123478/105868986-0cb9dc00-5ff7-11eb-867a-313458cb0039.png) | ![Screenshot 2021-01-26 at 16 52 08](https://user-images.githubusercontent.com/4123478/105869006-12172680-5ff7-11eb-9ec2-c5ff3ff09c49.png) | ![image](https://user-images.githubusercontent.com/4123478/105919445-f2ebb980-6035-11eb-8f28-4c33bc11a26f.png) |
